src/style.css

```
@@ -1 +1,30 @@
 /* add styles here */
+
+* {
+  margin: 0;
+}
+
+.stars__star{
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:first-of-type,
```

```suggestion
.stars--1 .stars__star:first-child,
```
Not a mistake though) but it would be more consistent)